docs/collections/_blueprints/general-aa-meta-tags.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: project-top
+date: 2024-05-03
+title: "Asset Analysis Meta Tag Application"
+description: "<strong>Asset Analysis Meta Tag Application</strong>"
+extract: "A Asset labelling system that allows the user to assign additional attributes to existing Asset that aid in data analysis methods."
```

Comment:
*An Asset

*to an existing Asset

* that aids

Reply:
done